### PR TITLE
Using local tslint

### PR DESCRIPTION
**Pull request by arusakov**
- Try using local tslint package instead of prebuilt.
- Add useLocalTslint option.
- Fix some coffeelint warnings.
- Clear code.

### CODE REVIEW

**Comment by park9140** [comment]
Any reason not to make this default?

**Comment by arusakov** [comment]
@park9140 No reason. Enable useLocalTslint by default.

File: lib/init.coffee [hunk 1]
```diff
@@ -9,42 +11,83 @@ module.exports =
...
       default: ''
+    useLocalTslint:
+      type: 'boolean'
+      title: 'Try using the local tslint package (if exist)'
+      default: true
+
+  rulesDirectory: ''
+  tslintCache: new Map
+  tslintDef: null
+  useLocalTslint: true
```
Comment (Arcanemagus) [inline lib/init.coffee]: No need for this, the `observe` in activate will set it for you when the plugin is activated.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "linter-tslint",
   "main": "./lib/init",
   "linter-package": true,
-  "version": "0.6.0",
+  "version": "0.7.0",
```
Comment (Arcanemagus) [inline package.json]: Don't bump the version here, `apm publish` does this automatically.

**Comment by Arcanemagus** [comment]
LGTM other than the two notes above, one of which isn't that important at all :wink:.

File: lib/init.coffee (hunk 1 above)
Comment (Arcanemagus) [inline lib/init.coffee]: This does help readability though, so you might want to just leave it in.

**Comment by arusakov** [comment]
@Arcanemagus Thx for review. I left ```0.6.0``` version.

**Comment by Arcanemagus** [comment]
@park9140 Any further changes?

**Comment by park9140** [comment]
:shipit:  I'll publish this tonight if no one gets to it first @archmagus.

**Pull request merged**